solcjs

```diff
@@ -14,12 +14,26 @@ var smtsolver = require('./smtsolver.js');
 process.removeAllListeners('uncaughtException');
 var commander = require('commander');
 
+const parseOptimizeRuns = function(value) {
+  const parsedValue = parseInt(value, 10);
+  if (isNaN(parsedValue)) {
+    return value;
+  }
+  return parsedValue;
+};
+
 var program = new commander.Command();
 program.name('solcjs');
 program.version(solc.version());
 program
   .option('--version', 'Show version and exit.')
   .option('--optimize', 'Enable bytecode optimizer.', false)
+  .option(
+    '--optimize-runs <optimize-runs>',
```

Comment:
It would be nice if this description here matches the main `solc` CLI:
>   --optimize-runs n (=200)
>                       Set for how many contract runs to optimize. Lower values
>                       will optimize more for initial deployment cost, higher 
>                       values will optimize more for high-frequency usage.

Reply:
I think the version in this PR is nicer, so I would suggest to create a PR against https://github.com/ethereum/solidity to update the other tool as well.

Reply:
Yeah, I thought that too, which is why I only suggested adding part of the description from solc.

Syncing it the other way seems like a good idea.